activerecord/lib/active_record/associations/builder/collection_association.rb

```diff
@@ -66,17 +66,5 @@ def #{name.to_s.singularize}_ids=(ids)
         end
       CODE
     end
-
-    def self.wrap_scope(scope, mod)
```

Comment:
What I mean is that now the scope defined by the association is not extending anymore the class since `extending` is not being called in the scope.